core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1403,31 +1432,66 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
         )
       }
 
+    def getRawlsPolicies(workspaceId: UUID): Future[Set[(WorkbenchEmail, SamResourcePolicyName)]] =
```

Comment:
`getRawlsPolicies` + `getWsmPolicies` both expose the same interface, there may be an abstraction to be had here -- `WorkspacePolicyManager` or some such thing, which would make this a bit more testable. I think this is probably fine for now though.